main.md

```diff
@@ -460,17 +462,18 @@ grant_type=authorization_code
 !---
 Figure: Token Request for a DPoP sender-constrained token using an authorization code {#token-request-code}
 
-The `DPoP` HTTP header MUST contain a valid DPoP proof JWT.
```

Comment:
Why is this line being deleted?

Reply:
`DPoP` header is defined above. iiuc this repeats the  MUST already present  in line 435.